site/learn/expert-advice/oauth/whats-new-with-oauth-2-1.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: advice
+title: What is new in the OAuth 2.1?
+description: A draft of the OAuth 2.1 specification was recently released. What's coming down the pike?
+image: advice/oauth-device-authorization-article.png
+author: Dan Moore
+category: OAuth
+date: 2020-10-20
+dateModified: 2020-10-20
```

Comment:
Make sure we have the correct redirect in place. This post gets a lot of traffic.

Reply:
I wasn't going to redirect. This is similar to what we did here: https://github.com/FusionAuth/fusionauth-site/pull/199/

So was planning to not even add a link on the blog post to this expert advice, just let it be present in the expert advice area and bulk that up some.

Reply:
My concern with doing it this way is being penalized for duplicate content

Reply:
I understand. I thought that if we changed ~10% of the content we wouldn't have to worry about that.

Reply:
```
old: 3136 words  2468 78% common  134 4% deleted  534 17% changed
new: 3264 words  2468 75% common  212 6% inserted  584 17% changed
```